packages/extension-tile-playground/README.md

```diff
@@ -0,0 +1,109 @@
+# Extension Tile Playground
+
+## Purpose
+
+The extension tile playground is a simple extension that allows changes to the extension SDKs to be
+quickly prototyped and tested in dashboard mount points. Do not save prototyping to this repo as this
+extension should remain simple. Instead, demos of extension SDK changes should be added to the
+[extension examples repo](https://github.com/looker-open-source/extension-examples).
+
+## Using the Extension Tile Playground
+
+The extension tile playground can be manually installed and run with a Looker instance using the
+following steps:
+
+1. create a new LookML project called `tile_extensions`
+1. create a new model. In `tile_extensions.model`, put:
+   ```lookml
+   connection: "<any valid connection name>"
+   ```
+1. in `manifest.lkml` put:
+
+   ```lookml
+   project_name: "tile_extensions"
+
+   application: vis {
+      label: "Visualization"
+      # file: "bundle.js"
+      url: "https://localhost:8080/dist/bundle.js"
+      mount_points: {
+         dashboard_vis: yes
+      }
+      entitlements: {
+         local_storage: yes
+         use_form_submit: yes
+         core_api_methods: []
+         external_api_urls: []
+         oauth2_urls: []
+         scoped_user_attributes: []
+         global_user_attributes: []
+      }
+   }
+
+   application: tile {
+      label: "Tile"
+      # file: "bundle.js"
+      url: "https://localhost:8080/dist/bundle.js"
+      mount_points: {
+         dashboard_tile: yes
+      }
+      entitlements: {
+         local_storage: yes
+         use_form_submit: yes
+         core_api_methods: []
+         external_api_urls: []
+         oauth2_urls: []
+         scoped_user_attributes: []
+         global_user_attributes: []
+      }
+   }
+   ```
+
+1. save all changes and deploy to production
+1. in the root of `sdk-codegen`:
+   ```sh
+   yarn && yarn build
+   ```
+1. in `packages/extension-tile-playground`:
+   ```sh
+   yarn develop
```

Comment:
Didn't you create a `yarn dev:play` alias for this?

Reply:
I did. A changed for this will be in the next commit